Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -3384,10 +3388,9 @@ - (void)uiDatabaseDidUpdate:(NSNotification *)notification
         // These errors seems to be very rare; they can only be reproduced
         // using the more extreme actions in the debug UI.
         OWSProdLogAndFail(@"%@ hasMalformedRowChange", self.logTag);
-        [self reloadViewItems];
-        [self.collectionView reloadData];
-        self.lastReloadDate = [NSDate new];
+        [self resetContentAndLayout];
```

Comment:
DRY this up into `resetContentAndLayout`.  Note that we were neglecting to invalidate the layout here.  Shouldn't have mattered since I don't know that we're hitting this case in production. This check is defensive.